docs/csharp/fundamentals/functional/discards.md

```diff
@@ -29,7 +29,7 @@ Discards are useful in working with tuples when your application code uses some
 
 For more information on deconstructing tuples with discards, see [Deconstructing tuples and other types](deconstruct.md#tuple-elements-with-discards).
 
-The `Deconstruct` method of a class, structure, or interface also allows you to retrieve and deconstruct a specific set of data from an object. You can use discards when you're interested in working with only a subset of the deconstructed values. The following example deconstructs a `Person` object into four strings (the first and last names, the city, and the state), but discards the last name and the state.
+The `Deconstruct` method of a class, structure, or interface also allows you to retrieve and deconstruct a specific set of data from an object. You can use discards when you're interested in working with only a subset of the deconstructed values. The following example deconstructs a `Person` object into four strings (the first and last names, the city, and the state), but discards the middle name and the state.
```

Comment:
That doesn't seem to be the correct change.

The example from [the article](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards#tuple-and-object-deconstruction) (at the end of the section):
![image](https://user-images.githubusercontent.com/15279990/235237721-74302b13-f180-449d-9b74-efff52104f4f.png)

Deconstructor is defined to output the last name, not middle. So, when in use:
![image](https://user-images.githubusercontent.com/15279990/235238092-74f3fff4-c6b9-4c4c-bb53-f4d092389990.png)

the last name is discarded, not middle.

Reply:
I agree with @pkulikov 

This looks like a mistake. I'm inclined to close this without merging.

Can you tell us what you were trying to fix @daleardi ?